Dockerfile

```diff
@@ -0,0 +1,19 @@
+# For local development purposes only
```

Comment:
Is this comment obvious and dumb to include? I added it because there is a line in here that removes the requirement for https, and I don't want anyone to think they can use this dockerfile to build a production server without removing that.

Reply:
I appreciate forward-looking CYA and don't consider this dumb at all.